src/components/ProgressSection.jsx

```diff
@@ -0,0 +1,27 @@
+import { ValidationStrategy as PropTypes } from "@eslint/object-schema";
+
+export default function ProgressSection({ applicationsCount, total }) {
+  const ratio = applicationsCount / total;
+
+  return (
+    <div className="mt-4">
+      <p className="text-base font-light mb-2">Progress</p>
```

Comment:
Maybe use `h6` tag instead of `p` tag as it matches the text weight and font family from the designs
```suggestion
      <h6 className="mb-2">Progress</h6>
```